lib/os-cluster-entrypoint.ts

```diff
@@ -249,30 +249,30 @@ export class OsClusterEntrypoint {
         securityDisabled: security,
         opensearchVersion: distVersion,
         clientNodeCount: clientCount,
-        cpuArch,
+        cpuArch: cpuArch,
```

Comment:
Does it make any difference?
Since passed argument name and accepting parameter name is same, having `parameter1: argument1` is same as `parameter1`. Or is it different?

Reply:
I don't think variable name as any significance in mapping (I might be wrong). This is to keep the args consistent and not worry about naming or order of arguments (positional) since the list is only going to grow from now on.  Recommending to choose one for better maintenance and passing between classes.

Reply:
You are right! TS is weird. The compiler is resetting in my local too. Found this: https://stackoverflow.com/questions/49798156/typescript-variable-name-as-interface-type
Do you think it is a better idea to have different variable names than the keyword arguments to be passed onto as key value pairs ? If not you can close this PR @rishabh6788